src/_includes/side_nav.html

```diff
@@ -0,0 +1,35 @@
+{% assign parent_page = site.pages | parent_of: page %}
```

Comment:
I wonder if this file should be semantically named, in case the position continues to be different on small screens? `page-list` or `page-nav`?

Reply:
you are right - I have renamed to 'site_navigation' ?